### PR TITLE
Weblate translations (2018-10-26)

### PR DESCRIPTION
**Pull request by renatolond**

### CODE REVIEW

**Review by mayaeh — DISMISSED** [review]

**Comment by BoFFire** [comment]
Requested to review the deleted lines.

**Comment by BoFFire** [comment]
In the sametime, I’m reviewing all the languages now, via :  https://weblate.joinmastodon.org/checks/

**Comment by nightpool** [comment]
blocked on https://github.com/tootsuite/mastodon/pull/9104

**Comment by Gargron** [comment]
I have merged #9104. A lot of broken translations were fixed in that one, and a new CI check was added to ensure newly imported translations aren't broken in the same ways. If you can, please rebase this PR and make sure the fixed things aren't overwritten. If you don't have time, let me know so I can take over.

**Comment by renatolond** [comment]
Hi! I will have some time probably tonight or tomorrow morning to check this one out :)

**Comment by BoFFire** [comment]
Finished to mark +3000 strings as needed to be edited. Now if you want to do a*a nice job*, it’s time to push from Weblate to Git and not the other way.

https://weblate.joinmastodon.org/checks/?project=mastodon

Stats : https://weblate.joinmastodon.org/stats/

Have a nice day.

**Comment by renatolond** [comment]
I've tried to look #9100 and #9104 and remove stuff that was reintroduced, I think most of it was gone, I need to review the `cy.yml`, I think there's still some stuff there.
About the removed ones, I think the unused task seems to be ignoring it, it was removed in .en and it was not detected by the task, I had to remove it by hand.
I'm going to take a look at those plurals.

File: config/locales/cy.yml [hunk 1]
```diff
@@ -481,77 +544,377 @@ cy:
...
     proceed: Cadw
+    updated_msg: Diweddarwyd gosodiad mudo eich cyfrif yn llwyddiannus!
   moderation:
     title: Cymedroli
   notification_mailer:
+    digest:
+      action: Gweld holl hysbysiadau
+      body: Dyma grynodeb byr o'r holl negeseuon golloch chi ers eich ymweliad diwethaf ar %{since}
+      mention: 'Soniodd %{name} amdanoch chi:'
+      new_followers_summary:
```
Comment (mayaeh) [inline config/locales/cy.yml]: Maybe I think that need to organize 627-628, 630-631 lines as well.

It has not been pointed out in # 9104, but it has been pointed out in cs.yml .
https://github.com/tootsuite/mastodon/pull/9104/files#diff-2a5406712ed4bdf998f81b4ae9fb3ae1R615

Does anyone know about it?

File: config/locales/cy.yml [hunk 2]
```diff
@@ -481,77 +544,377 @@ cy:
...
+    your_apps: Eich rhaglenni
+  statuses:
+    attached:
+      description: 'Ynghlwm: %{attached}'
+      image:
+        one: "%{count} llun"
+        other: "%{count} o luniau"
+      video:
+        one: "%{count} fideo"
+        other: "%{count} fideo"
```
Comment (mayaeh) [inline config/locales/cy.yml]: Also here.
I'm sorry I can not judge whether there should be changes in singular and plural form.

**Comment by renatolond** [comment]
No matter how I change the translation file, it seems `zh-TW` continues to complain about plurals.

`zh-TW  | followers.success                                | zero, two, few, many`

If I add `zero, two, few, many`, it complains about `other`. If I add `other`, it complains again about `zero, two, few, many`, even they being there.

**Comment by renatolond** [comment]
I merged the latest changes from weblate, but I don't know to fix `zh-TW` issues.

**Comment by Gargron** [comment]
@renatolond Did you also merge latest changes from master? Might have the old i18n-tasks that has the wrong locale bug

**Comment by renatolond** [comment]
@Gargron Yes, the only commit that's not in the branch from master is the latest one (the dimensions of preview cards one)

**Comment by Gargron** [comment]
@renatolond Could you add this commit?

```patch
From 0bb409b826654dc704df1eb9db45c197828729da Mon Sep 17 00:00:00 2001
From: Eugen Rochko <eugen@zeonfederated.com>
Date: Mon, 29 Oct 2018 13:03:47 +0100
Subject: [PATCH 182/182] Fix cy locale


---
config/locales/cy.yml | 28 +++++++---------------------
1 file changed, 7 insertions(+), 21 deletions(-)


diff --git a/config/locales/cy.yml b/config/locales/cy.yml
index 0b836ab..a4df28b 100644
--- a/config/locales/cy.yml
+++ b/config/locales/cy.yml
@@ -558,9 +558,7 @@ cy:
     followers_count: Nifer y dilynwyr
     lock_link: Cloi eich cyfrif
     purge: Dileu o dilynwyr
-    success:
-      one: Yn y broses o ysgafn-flocio dilynwyr o un parth...
-      other: Yn y broses o ysgafn-flocio dilynwyr o %{count} parth...
+    success: Yn y broses o ysgafn-flocio dilynwyr o %{count} parth...
     true_privacy_html: Cofiwch <strong>mai ond amgryptio pen-i-ben all sicrhau gwir breifatrwydd</strong>.
     unlocked_warning_html: Gall unrhywun eich dilyn yn syth i weld eich tŵtiau preifat. %{lock_link} i gael adolygu a gwrthod dilynwyr.
     unlocked_warning_title: Nid yw eich cyfrif wedi ei gloi
@@ -595,9 +593,7 @@ cy:
     expires_in_prompt: Byth
     generate: Cynhyrchu
     invited_by: 'Cawsoch eich gwahodd gan:'
-    max_uses:
-      one: 1 defnydd
-      other: "%{count} defnydd"
+    max_uses: "%{count} defnydd"
     max_uses_prompt: Dim terfyn
     prompt: Cynhyrchwch a rhannwch ddolenni gyda eraill i ganiatau mynediad i'r achos hwn
     table:
@@ -623,12 +619,8 @@ cy:
       action: Gweld holl hysbysiadau
       body: Dyma grynodeb byr o'r holl negeseuon golloch chi ers eich ymweliad diwethaf ar %{since}
       mention: 'Soniodd %{name} amdanoch chi:'
-      new_followers_summary:
-        one: Hefyd, rydych wedi ennill dilynwr newydd tra eich bod i ffwrdd! Hwrê!
-        other: Hefyd, rydych wedi ennill %{count} dilynwr newydd tra eich bod i ffwrdd! Hwrê!
-      subject:
-        one: "1 hysbysiad newydd ers eich ymweliad diwethaf \U0001F418"
-        other: "%{count} hysbysiad newydd ers eich ymweliad diwethaf \U0001F418"
+      new_followers_summary: Hefyd, rydych wedi ennill %{count} dilynwr newydd tra eich bod i ffwrdd! Hwrê!
+      subject: "%{count} hysbysiad newydd ers eich ymweliad diwethaf \U0001F418"
       title: Yn eich absenoldeb...
     favourite:
       body: 'Cafodd eich statws ei hoffi gan %{name}:'
@@ -745,17 +737,11 @@ cy:
   statuses:
     attached:
       description: 'Ynghlwm: %{attached}'
-      image:
-        one: "%{count} llun"
-        other: "%{count} o luniau"
-      video:
-        one: "%{count} fideo"
-        other: "%{count} fideo"
+      image: "%{count} o luniau"
+      video: "%{count} fideo"
     boosted_from_html: Wedi ei fŵstio %{acct_link}
     content_warning: 'Rhybudd cynnwys: %{warning}'
-    disallowed_hashtags:
-      one: 'yn cynnwys hashnod gwaharddedig: %{tags}'
-      other: 'yn cynnwys yr hashnod gwaharddedig: %{tags}'
+    disallowed_hashtags: 'yn cynnwys yr hashnod gwaharddedig: %{tags}'
     language_detection: Canfod iaith yn awtomataidd
     open_in_web: Agor yn y wê
     over_character_limit: wedi mynd heibio'r uchafswm nodyn o %{max}
--
2.7.4
```

**Comment by Gargron** [comment]
I think what happened is that since I updated the i18n-dependency to my own PR branch to fix the incorrect "other" listing, we no longer had the "wrong locale" fix which was in a different PR. Now both are in i18n-tasks's master branch, so updating to that is probably wise.

**Comment by renatolond** [comment]
Applied the diff!
Let me see if updating to i18n-tasks master solves the issue

**Review by Gargron — APPROVED** [review]

**Comment by renatolond** [comment]
Yeah, you're right, with master it shows cy and not zh-tw as the source of the issues :)
Do you prefer to wait for a release of i18n-tasks or to point to the master?

**Comment by Gargron** [comment]
I carelessly deleted my PR branch so fetching bundler dependencies fails currently, so updating to i18n-tasks master would be safer. We might hopefully get a full release of i18n-tasks today though...

**Pull request merged**